datalad/distribution/tests/test_create.py

```diff
@@ -349,3 +350,19 @@ def test_create_text_no_annex(path):
     ds.add(['t', 'b'])
     ok_file_under_git(path, 't', annexed=False)
     ok_file_under_git(path, 'b', annexed=True)
+
+
+@with_tempfile(mkdir=True)
+def test_create_fake_dates(path):
+    ds = create(path, fake_dates=True)
+
+    eq_(ds.config.get("datalad.dataset.fakedates"), "true")
```

Comment:
> may be to stay "consistent" with fake_dates should become datalad.dataset.fake-dates here?

The hyphen feels wrong to me since Git doesn't use hyphen in their config variable names, but I'll switch it if you prefer.

> also I am not sure if it should be under .dataset space of some kind to indicate that we are faking dates within dataset, or just as datalad.fake-dates?

I was wondering that too :]  Didn't go with ".dataset" for any good reason.

Reply:
> Also should gets its record within [...]

Thanks, will add.

Reply:
looking at
```
[datalad "dataset"]
    fakedates = true
[annex]
    uuid = c09e667a-b759-4fa4-9142-a7614b214d9b
```
I think that we should definitely strip away `.dataset` level